src/main/java/com/dynatrace/openkit/core/util/InetAddressValidator.java

```diff
@@ -26,32 +26,53 @@
 
     private static final Pattern IPV4_PATTERN =
         Pattern.compile(
-            "^(25[0-5]|2[0-4]\\d|[0-1]?\\d?\\d)(\\.(25[0-5]|2[0-4]\\d|[0-1]?\\d?\\d)){3}$");
+            "^"                                             // start of string
+          + "(25[0-5]|2[0-4]\\d|[0-1]?\\d?\\d)"             // first block - a number from 0-255
+          + "(\\.(25[0-5]|2[0-4]\\d|[0-1]?\\d?\\d)){3}"     // three more blocks - numbers from 0-255 - each prepended by a point character '.'

```

Comment:
In context of IP addresses, I'd personally use 'dot' instead of 'point', but I'm no native speaker...